server/src/main/java/io/seata/server/storage/redis/LuaParser.java

```diff
@@ -149,4 +157,14 @@ public static <T> List<T> getListFromJson(String json, Class<T> classz) {
             throw new StoreException(e.getMessage());
         }
     }
+
+    public static Object jedisEvalSha(Jedis jedis, String luaSHA, String luaFileName, List<String> keys, List<String> args){
+        try {
+            return jedis.evalsha(luaSHA, keys, args);
+        }catch (JedisNoScriptException e) {
+            LOGGER.warn("jedis ex: " + e.getMessage());
```

Comment:
这里日志要提示一下，否则用户会认为异常了，但实际上我们继续了reload script和重新执行了lua脚本
Here the logs should be prompted, otherwise the user will think it's an exception, but in fact we continue the reload script and re-execute the lua script.

Reply:
done